### ISSUE TITLE
WEEK 6 ANSWERS ISSUE

### ISSUE THREAD

**Issue by Karthik4111** [salimt/Courses-#38]
1)
Run your program with the earthquake input file "quiz2.atom". There is a commented out line of code in the setUp method in EarthquakeCityMap that you can uncomment to make this happen:

Call sortAndPrint as appropriate to answer the following question.

What is the magnitude of the 6th earthquake printed (i.e. the 6th strongest earthquake)? To be clear, if there are two earthquakes of the same magnitude, each magnitude will count. For example, if the top 4 magnitudes are {7, 6.9, 6.9, and 6}, the 4th largest is 6

May I Know this Answer Please ?Which is in 6th week of assignment.Please Help me.

2)
Run your program with the earthquake input file "quiz2.atom". There is a commented out line of code in the setUp method in EarthquakeCityMap that you can uncomment to make this happen:
Call sortAndPrint as appropriate to answer the following question.

What is the strongest magnitude that is repeated three or more times in the data set?

May I Know this Answer Please ?Which is in 6th week of assignment.Please Help me.

**Comment by KushagraBhargav** [comment]
did u find the answers?

**Comment by Saif-2000** [comment]
> Run your program with the earthquake input file "quiz2.atom". There is a commented out line of code in the setUp method in EarthquakeCityMap that you can uncomment to make this happen:
> 
> Call sortAndPrint as appropriate to answer the following question.
> 
> What is the magnitude of the 6th earthquake printed (i.e. the 6th strongest earthquake)? To be clear, if there are two earthquakes of the same magnitude, each magnitude will count. For example, if the top 4 magnitudes are {7, 6.9, 6.9, and 6}, the 4th largest is 6
> 
> May I Know this Answer Please ?Which is in 6th week of assignment.Please Help me.
> 
> 
> Run your program with the earthquake input file "quiz2.atom". There is a commented out line of code in the setUp method in EarthquakeCityMap that you can uncomment to make this happen:
> Call sortAndPrint as appropriate to answer the following question.
> 
> What is the strongest magnitude that is repeated three or more times in the data set?
> 
> May I Know this Answer Please ?Which is in 6th week of assignment.Please Help me.

**Comment by rosy-swarnali** [comment]
Who are you?? Why would I help you?

**Comment by sakibkaderchowdhury** [comment]
i am also struggling with this 2 questions, it would be very helpful if someone could help us ^^

**Comment by pjajagtap** [comment]
need help in same questions

**Comment by sushmithaswaraj** [comment]
Need help in same questions

**Comment by sushmithaswaraj** [comment]
Pls help me

**Comment by sushmithaswaraj** [comment]
Need help in same question pls help me

**Comment by sakibkaderchowdhury** [comment]
hello,
sorry could not reply to you so could you be specific like i think you want
week 6 object oriented programming java answer?
if u need anything else in that course let me know
thank you :)

**Comment by sushmithaswaraj** [comment]
Yes when i recieve the answers of week-6 object oriented in java

**Comment by sakibkaderchowdhury** [comment]
hello
i could not understand clearly :3
So should I send the answers or not?

**Comment by sushmithaswaraj** [comment]
Sir plz help me

**Comment by reyanbhuiyan** [comment]
anyone know this 2 question ans ?

**Comment by sakibkaderchowdhury** [comment]
1-1
2-4
3-1
4-2
5-1
6-4
7-6.4
8-5.9
9-ur words
10-ur words

**Comment by sakibkaderchowdhury** [comment]
it's high>=low

**Comment by Subhee1** [comment]
> need help in same questions

**Comment by sakibkaderchowdhury** [comment]
do u still need help with this? should i send the answers?

**Comment by SimonSongCA** [comment]
Hi everybody,
Is there such kind of a platform so that we could keep in touch with everybody whenever there is an issue of Coursera? We could stick on that platform so as to have in-time responses!

**Comment by imranmohamed1986** [comment]
I am also struck with those two questions.does anyone get the answer? really helpful if you helped me, thanks.

**Comment by AhmadEdrisNaderi** [comment]
Question 7
Run your program with the earthquake input file "quiz2.atom".  There 
is a commented out line of code in the setUp method in EarthquakeCityMap
 that you can uncomment to make this happen

**Comment by AhmadEdrisNaderi** [comment]
can you pelase help me about this question ?7.
Question 7
Run your program with the earthquake input file "quiz2.atom".  There 
is a commented out line of code in the setUp method in EarthquakeCityMap
 that you can uncomment to make this happen

**Comment by sakibkaderchowdhury** [comment]
Hello, I moved to usa and almost forgot this things,
so sorry, could not help you.

**Comment by MustafaDemiroglu** [comment]
i have just seen it and it was funny

**Comment by Wayasel** [comment]
Question 8 Plz Help Me
Run your program with the earthquake input file "quiz2.atom".  There 
is a commented out line of code in the setUp method in EarthquakeCityMap
 that you can uncomment to make this happen: earthquakesURL = "quiz2.atom";  

Call sortAndPrint as appropriate to answer the following question.
What  is the strongest magnitude that is repeated three or more times in the data set?

**Comment by thanhtrongg** [comment]
> Question 8 Plz Help Me Run your program with the earthquake input file "quiz2.atom". There is a commented out line of code in the setUp method in EarthquakeCityMap that you can uncomment to make this happen: earthquakesURL = "quiz2.atom";
> 
> Call sortAndPrint as appropriate to answer the following question. What is the strongest magnitude that is repeated three or more times in the data set?

5.9
